### PR TITLE
Added functionality to enable/disable role scanner

### PR DESCRIPTION
**Pull request by red2k18**
Added functionality to enable/disable role scanner.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -553,6 +553,12 @@ variable "resource_enabled" {
   default     = true
 }
 
+variable "role_enabled" {
+  description = "Role scanner enabled."
+  type        = bool
+  default     = true
```
Comment (gkowalski-google) [inline variables.tf]: Since there aren't any default rules for the Role scanner, should we disable it by default?

Comment (red2k18) [reply variables.tf]: Yes, disabled Roles scanner.

**Comment by gkowalski-google** [comment]
Resolves #522

**Pull request merged**